packages/flutter/lib/src/material/data_table.dart

```diff
@@ -370,6 +370,14 @@ class DataCell {
 /// ```
 /// {@end-tool}
 ///
+/// [DataTable] can be sorted on the basis of any column in [columns] in
+/// ascending or descending order. If [sortColumnIndex] is mentioned then that
```

Comment:
I think this could be said a little more briefly:

If [sortColumnIndex] is non-null, then the table will be sorted by the values in the specified column. The boolean [sortAscending] flag controls the sort order.

Reply:
Sue @HansMuller I will change it.